packages/edit-navigation/src/components/inspector-additions/style.scss

```diff
@@ -0,0 +1,20 @@
+// Set the block inspector to use flexbox so that we can change the order of
+// panels within it.
+// TODO: Is there a way to do this natively using InspectorControls?
+.block-editor-block-inspector {
+	display: flex;
+	flex-direction: column;
+
+	.block-editor-block-card {
+		order: -2;
```

Comment:
Oooh we shouldn't do this, it messes up keyboard focus order.
I don't think there's a way to add items to the inspector in any position, because the slot is positioned in a specific place; perhaps the component could be changed to include another slot, but... do we really need the top level pages checkbox to be this prominent?

Reply:
Yeah, this is a total hack. We need to come up with a better way of doing this. Maybe we could add an `order` prop to `InspectorControls`. I've made a note to come back to this.